tensorflow_datasets/image/food101.py

```diff
@@ -0,0 +1,79 @@
+"""Dataset class for Food-101 dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+import os
+
+_BASE_URL = "http://data.vision.ee.ethz.ch/cvl/food-101.tar.gz"
+
+_DESCRIPTION = ("This dataset consists of 101 food categories, with 101'000 images. For each class, "
+"250 manually reviewed test images are provided as well as 750 training images. On purpose, the training images were not cleaned, "
+"and thus still contain some amount of noise. This comes mostly in the form of intense colors and sometimes wrong labels. "
+"All images were rescaled to have a maximum side length of 512 pixels.")
+
+_LABELS_FNAME = 'image/food-101_classes.txt'
+
+_CITATION = """\
+
+ @inproceedings{bossard14,
+  title = {Food-101 -- Mining Discriminative Components with Random Forests},
+  author = {Bossard, Lukas and Guillaumin, Matthieu and Van Gool, Luc},
+  booktitle = {European Conference on Computer Vision},
+  year = {2014}
+}
+
+"""
+
+class Food101(tfds.core.GeneratorBasedBuilder):
+  """Food-101 Images dataset"""
+
+  VERSION = tfds.core.Version('1.0.0')
+
+  def _info(self):
+        """Define Dataset Info"""
+        names_file = tfds.core.get_tfds_path(_LABELS_FNAME)
+        return tfds.core.DatasetInfo(
+            builder=self,
+    
+            description=(_DESCRIPTION),
+            
+            features=tfds.features.FeaturesDict({
+                "image":tfds.features.Image(),
+                "label": tfds.features.ClassLabel(names_file=names_file),
+            }),
+
+            supervised_keys=("image", "label"),
+            
+            urls=[_BASE_URL],
```

Comment:
you should add dataset [link](https://www.vision.ee.ethz.ch/datasets_extra/food-101/) not a download link.